operations/overlay_fisher_exact_test.yml

```diff
@@ -0,0 +1,42 @@
+id: overlay_fisher_exact_test
+subclass_of: overlay
+name: Overlay Fisher exact test
+description: Fisher exact test computes the Fisher's Exact Test p-values of the connection between a list of given nodes with specified query id (subject_qnode_key e.g. n01) to their adjacent nodes with specified query id (object_qnode_key e.g. n02) in the message knowledge graph. This information is then added as an edge attribute to a virtual edge which is then added to the query graph and knowledge graph. It can also allow you to filter out the user-defined insignificance of connections based on a specified p-value cutoff or return the top n smallest p-value of connections and only add their corresponding virtual edges to the knowledge graph.
+examples: 
+  - input: overlay/messages/11_input_fisher.json
+    output: overlay/messages/12_output_fisher.json
+    parameters:
+      virtual_relation_label: f1
+      subject_qnode_key: "protein" 
+      object_qnode_key: "type-2 diabetes"
+input_requirements: []
+output_guarantees: []
+allowed_changes:
+  - add kedges
+  - add qedges
+  - add bindings to results
+
+parameters:
+  type: object
+  properties:
+    subject_qnode_key:
+      description: A specific subject query node id.
+      type: string
+      example: n1
+    object_qnode_key:
```

Comment:
Are subject and object interchangeable here?

Reply:
They are not. Swapping them with give different answers. You are checking if the object node is enriched by the subject node.